src/ontology/config/properties.txt

```diff
@@ -15,6 +15,7 @@ http://www.geneontology.org/formats/oboInOwl#source
 http://www.geneontology.org/formats/oboInOwl#hasSynonymType
 http://www.geneontology.org/formats/oboInOwl#SynonymTypeProperty
 http://purl.obolibrary.org/obo/mondo#GENERATED
+http://purl.obolibrary.org/obo/mondo#omim_included
```

Comment:
## The change
I reran `components/omim.owl` and it's showing up as expected, e.g. (snippet):

```owl
    <owl:Class rdf:about="https://omim.org/entry/100300">
        <mondo:omim_included>aplasia cutis congenita, congenital heart defect, and frontonasal cysts</mondo:omim_included>
        <oboInOwl:hasExactSynonym>AOS1</oboInOwl:hasExactSynonym>
        <skos:exactMatch rdf:resource="http://purl.obolibrary.org/obo/MONDO_0024506"/>
    </owl:Class>
```

I guess to test for unintended side effects, the build should e run even for small cases like this, so I'll set that running tonight.